### PR TITLE
Update __init__.py

### LINKED ISSUE #91080: Torch.optim Errata
### 📚 The doc issue

In “To use torch.optim you have to construct an optimizer object, that will hold the current state and will update the parameters based on the computed gradients”, change “object, that” to “object that”.

In “can be also easily integrated”, change “be also” to “also be”.

### Suggest a potential alternative/fix

_No response_

cc @svekars @carljparker

### PR DESCRIPTION
**Pull request by richardachen**
Fixes #91080

### CODE REVIEW

**Review by janeyx99 — COMMENTED** [review]
Will you also be including the other rephrasing mentioned in the issue in this PR?

These are both small/relevant enough to be landed as one PR.

**Comment by richardachen** [comment]
Sorry about that - I had the other rephrasing suggestion fixed through another PR. This is the remaining suggestion.

**Review by janeyx99 — APPROVED** [review]
oh i see, no worries!
